consensus/module.go

```diff
@@ -27,8 +26,15 @@ var _ modules.ConsensusModule = &consensusModule{}
 type consensusModule struct {
 	bus        modules.Bus
 	privateKey cryptoPocket.Ed25519PrivateKey
+        // For BLS signature aggregation
+        var sec bls.SecretKey
```

Comment:
Does this compile?

I don't think variable declaration and variable assignment inside a struct definition works.

Reply:
I think it may have not, restructured since.